dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -191,6 +190,19 @@ public WorkflowResource(HttpClient client, SessionFactory sessionFactory, Permis
         }
     }
 
+
+    public EntryResource getEntryResource() {
```

Comment:
The only thing that seems to call this is within this class?

Reply:
Yes; I can remove the accessor function; let me know

Reply:
Removed the accessor function